addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23940,7 +23940,37 @@ msgctxt "#40032"
 msgid "No similar movies were found in the library."
 msgstr ""
 
-#empty strings with id 40033 to 40199
+#. Addition of new version: a movie with multiple versions cannot be turned into a version of another movie.
+#: xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp
+msgctxt "#40033"
+msgid "The default version of a movie with multiple versions cannot be added as a version to another movie. Please add its additional versions first."
+msgstr ""
+
+#. Addition of new extra: a movie with multiple versions cannot be turned into an extra of another movie.
+#: xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp
+msgctxt "#40034"
+msgid "The default version of a movie with multiple versions cannot be added as an extra to another movie. Please move or remove its versions first."
+msgstr ""
+
+#. Addition of new version: the user picked an extra of another movie. Ask for confirmation.
+#: xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp
+msgctxt "#40035"
+msgid "You are about to convert a movie extra into a movie version. Are you sure?"
+msgstr ""
+
+#. Addition of new version: the user picked an extra of another movie. Ask for confirmation.
+#: xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp
+msgctxt "#40036"
+msgid "You are about to convert a movie version into a movie extra. Are you sure?"
+msgstr ""
+
+#. Addition of new version to a movie: the user chose a movie that has multiple versions. Ask for confirmation.
+#: xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp
+msgctxt "#40037"
+msgid "The movie to be added has multiple versions.[CR]The type of the default version will be selected in the next dialog. The other versions will keep their current type.[CR]Do you want to continue?"
```

Comment:
I think a dedicated dialog would be needed to make things visual and simplify or eliminate the message. You'll notice that it doesn't even have variables to make it clearer by spelling out the movie names and version types referred to, but that would make it unmanageably long because the yes/no dialog is small.
Regarding CR, this message is extra long, 3 sentences and new. Try for yourself without CR, I'm not opposed to a revision and welcome the feedback. The other messages are 2 sentences and variations of existing messages without CR. There is a balance to strike because CR may add a line, which causes a text to scroll, which is worse for readability I believe...

Reply:
Yeah, i have no idea for better strings. So nothing i can advice, sorry.